src/main/java/com/courtcanva/ccfranchise/services/FranchiseeService.java

```diff
@@ -114,5 +121,29 @@ public Optional<Franchisee> findFranchiseeById(Long franchiseeId) {
 
     }
 
+    @Transactional
+    public OrderListGetDto acceptOrders(OrderListPostDto orderListPostDto) {
+
+        List<Order> selectedOrders = orderRepository.findByIdIn(
+                orderListPostDto.getOrders()
+                        .stream()
+                        .map(OrderPostDto::getId)
+                        .collect(Collectors.toList()));
+
+        if (selectedOrders.isEmpty()) {
 
+            log.debug("selected order id: {} is empty", orderListPostDto.getOrders());
+            throw new ResourceNotFoundException("You have not selected any order.");
```

Comment:
zero accepted orders is normal situation, not good to throw exception

Reply:
Zero accepted orders may be a normal situation, but users may forget that they have not selected any orders and click the submit button. If I don't throw an exception maybe the user won't notice they forgot to select the order

Reply:
Yeah, agree with you.